include/rangers-api/Application/UI/UIMusicSelect.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+namespace app::ui {
+    class UIMusicSelect : public hh::game::GameObject, public hh::ui::UIListener, public hh::hid::ActiveDeviceManagerListener, public app::snd::CustomSoundListener {
+        hh::ui::LayerController* layerController;
+        hh::ui::LayerController* refBtn;
+        hh::ui::LayerController* refPcBtnT;
+        hh::ui::LayerController* refPcBtnB;
```

Comment:
These look like direct layer names from `.swif` files. Please name them properly or keep these type of variable names to comments.